applications/nrf_desktop/doc/usb_state.rst

```diff
@@ -6,3 +6,67 @@ USB state module
 The ``usb_state`` module is responsible for tracking the USB connection.
 It is also responsible for the data transmission that application does on this
 device.
+
+Module Events
+*************
+
++--------------------+----------------------------+-------------+-----------------------------------+------------------+
+| Source Module      | Input Event                | This Module | Output Event                      | Sink Module      |
++====================+============================+=============+===================================+==================+
++                    + ``module_state_event``     +             + ``module_state_event``            +                  +
++--------------------+----------------------------+             +-----------------------------------+------------------+
++ ``hid_state``      + ``hid_report_event``       +             + ``hid_report_sent``               + ``hid_state``    +
++--------------------+----------------------------+             +-----------------------------------+                  +
++                    + ``config_event``           +             + ``hid_report_subscription_event`` +                  +
++--------------------+----------------------------+             +-----------------------------------+------------------+
++                    + ``config_fetch_event``     +             + ``usb_state_event``               +                  +
++--------------------+----------------------------+             +-----------------------------------+------------------+
++                    + ``config_forwarder_event`` +             +                                   +                  +
++--------------------+----------------------------+-------------+-----------------------------------+------------------+
+
+
+Configuration
+*************
+
+The ``usb_state`` is enabled by selecting ``CONFIG_DESKTOP_USB_ENABLE``.
+It depends on :option:`CONFIG_USB_DEVICE_HID`.
+
+When enabling the USB support for the device the following generic options should be set correctly:
```

Comment:
```suggestion
When enabling the USB support for the device, set the following generic options:
```
"should be set correctly" -- what does it mean "correctly" here? Please add explanation next to every option, for example ` * :option:`CONFIG_USB_DEVICE_MANUFACTURER` - set the device manufacturer's name`

Reply:
Do you really think explaining it more is necessary? `the device manufacturer's name` seems pretty obvious the same goes with the other options. Or do you mean to add `set ...` there?